operatorhub-resources/local-testing/operator-source.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: rakhitharr-operators
+  namespace: marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: rakhitharr
+  
```

Comment:
What is this file? Can we remove ur name from here?

Reply:
So this is quay.io repo?

Reply:
The bundle image is currently in my quay.io repo.